etc/flameshot.profile

```diff
@@ -0,0 +1,38 @@
+# Firejail profile for flameshot
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/flameshot.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-interpreters.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+include /etc/firejail/disable-xdg.inc
```

Comment:
Maybe add `noblacklist ${PICTURES}` so screenshots can be saved in the pictures directory? If so, please add a blank line and then this line under `include /etc/firejail/globals.local`

Reply:
Smart - will add.